public/js/pimcore/object/tags/reverseObjectRelation.js

```diff
@@ -136,6 +104,7 @@ pimcore.object.tags.reverseObjectRelation = Class.create(pimcore.object.tags.man
 
         var className = record.data.text;
 
+        let columns = this.getVisibleColumns();
```

Comment:
```suggestion
        let columns = this.getVisibleColumns();
        
                if (!columns) {
            columns = [
                {text: 'ID', dataIndex: 'id', flex: 50},
                {text: t("reference"), dataIndex: 'fullpath', flex: 200, renderer: this.fullPathRenderCheck.bind(this)},
                {text: t("class"), dataIndex: 'classname', flex: 100}
            ];
        }
```

Reply:
@BlackbitDevs Could you please also set the `visibleFieldDefinitions`, otherwise the fields won't be displayed. Thanks https://github.com/pimcore/admin-ui-classic-bundle/blob/2da898db611eb8d05f893c48617ef2e71615ba5f/public/js/pimcore/object/tags/manyToManyObjectRelation.js#L362

Reply:
I think [this](https://github.com/pimcore/pimcore/pull/13569#discussion_r1342528482) change fixes the problem.

Reply:
@dvesh3 Thanks, approved